src/components/Book.jsx

```diff
@@ -17,22 +23,39 @@ const Book = ({ book }) => {
     return null; // or render a loading state
   }
   return (
-    <div>
-      <div key={book.item_id}>
-        <h3>{book.title}</h3>
-        <p>
-          Author:
-          {' '}
-          {book.author}
-        </p>
-        <p>
-          Category:
-          {' '}
+    <div key={book.item_id} className="book-card">
+      <div className="book-detail">
+        <p className="book-category">
           {book.category}
         </p>
-        <button type="button" onClick={handleRemoveBook}>
-          Remove
-        </button>
+        <span className="book-title">
+          {book.title}
+        </span>
+        <span className="author">{book.author}</span>
+        <div className="book-actions-container">
+          <button className="comments" type="button">Comments</button>
+          <div className="actions-separate-line" />
+          <button className="remove" type="button" onClick={handleRemoveBook}>
+            Remove
+          </button>
+          <div className="actions-separate-line" />
+          <button className="edit" type="button">Edit</button>
+        </div>
+      </div>
```

Comment:
- [ ] As I mentioned in my previous comment, you should follow the design properly, because as I can see you are not using the Correct `font-family`, `font-colour` etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the attached image and table and don't forget to align the items correctly.

### Requirement
![Screenshot from 2023-09-09 21-59-56](https://github.com/mulugithub/bookstore/assets/66476329/dbfd71d2-7a0d-47df-911c-07801c432843)

### Design Comparison
<table>
<tr>
<th>Zeplin's Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-09-09 22-02-19](https://github.com/mulugithub/bookstore/assets/66476329/3aaf1a69-224b-4a24-b6b4-c435c97f6ee6)

</td>
<td>

![Screenshot from 2023-09-09 22-02-28](https://github.com/mulugithub/bookstore/assets/66476329/fe38ec5a-e57f-4bcf-8f8b-bc68ba24622f)

</td>
</tr>
</table>